.github/workflows/build-pr-preview.yml

```diff
@@ -2,14 +2,7 @@ name: Build Preview
 run-name: 'Build Preview for "${{ github.event.pull_request.title }}" (#${{ github.event.pull_request.number }})'
 
 on:
-  pull_request_target:
-    types: [opened, synchronize]
-    branches:
-      - main
-      - develop
-  # paths:
-    # Hello Security 👋, we are checking to make sure forked repo PR changed paths are only in content/** inside the job security-check.
-    # We are doing this so we can also reuse this workflow for internal PRs, as pull_request_target also triggers on internal PRs. (As does pull_request)
+  push:
```

Comment:
@rmainwork because this is file is a `pull_request_target` on main, your changes in this PR will not go into effect in this PR. (This because security is bigger concern with `pull_request_target` actions.) Please duplicate this workflow file into a new push/pull_request workflow file so that you can see your test runs.

Reply:
Good point! I copied the workflow into another file, pushed it and (with lots of rebasing afterward) tested it to make sure it works - it seems like it does.